app/src/debug/res/values-de/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
for debug purpose only; to distinguish between regular KISS and debug build with my used locale 😉